scripts/bib2md.py

```diff
@@ -0,0 +1,209 @@
+"""
+Prints markdown for the Publications.md from .bib files.
+
+Usage: Call `main` with the path to the script and a list of citekeys.
```

Comment:
Maybe let's add a little bit of info on what `the script` is

Reply:
Well, "the script" is obviously a typo. It's supposed to read "the bibfile"